docs/source/torch.rst

```diff
@@ -555,5 +555,7 @@ Utilities
     promote_types
     set_deterministic
     is_deterministic
+    set_warnOnlyOnce
```

Comment:
cc @kurtamohler for these names, since we just updated the deterministic flags (see https://github.com/pytorch/pytorch/pull/49904)

Reply:
For the deterministic flag rename, we currently have `torch.use_deterministic_algorithms` and `torch.is_deterministic_algorithms_enabled` on the Python side, `Context::setDeterministicAlgorithms` and `Context::deterministicAlgorithms` on the C++ side.

I think an improvement might be to change `set_warnOnlyOnce` to `set_warn_only_once`.

If we wanted to be extremely strict about consistency with the deterministic flag, we might call it `use_warn_only_once`, but I don't think that grammar makes sense, so probably not a great idea.

Reply:
So for consistency I should replace all `warnOnlyOnce` with `warn_only_once` on both python and C++ level?

Reply:
ahh, I see. Later on the reviewers prefer the c++ names to be as is, and only change the python ones.